sonar-dotnet-shared-library/src/main/java/org/sonarsource/dotnet/shared/plugins/DotNetSensor.java

```diff
@@ -91,6 +91,8 @@ private void executeInternal(SensorContext context) {
     List<Path> protobufPaths = reportPathCollector.protobufDirs();
     if (!protobufPaths.isEmpty()) {
       protobufDataImporter.importResults(context, protobufPaths, toRealPath);
+    } else {
+      LOG.warn("No protobuf reports found - no metrics and highlighting will be imported.");
```

Comment:
@pavel-mikula-sonarsource  given that we fail if no roslyn reports are found... shouldn't we also (or even the more) fail if no highlighting can be made?

Reply:
I don't think so, I've chosen only the Roslyn property to fail fast because of this comment:
https://github.com/SonarSource/sonar-dotnet/blob/ee3a8953dacbde39b0ff18c86d04cb95724cd8a5/sonar-dotnet-shared-library/src/main/java/org/sonarsource/dotnet/shared/plugins/AbstractModuleConfiguration.java#L82

Having "SQ Project" containing only .NET test project and maybe some other non-.NET stuff you'd like to analyze is (weird but) legal setup with Roslyn in place and should be analyzed. This might change once `Tests as 1st class citizen` is done and we modify the pb generation and that comment.